apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -209,6 +237,20 @@ defmodule OMG.Watcher.ExitProcessor do
 
   use GenServer
 
+  @doc """
+  Initializes the state of the `ExitProcessor`'s `GenServer`.
+
+  Reads the exit data from `OMG.DB`.
+
+  Options:
+    - `exit_processor_sla_margin`: number of blocks after exit start before it's considered late (and potentially:
+      unchallenged)
+    - `exit_processor_sla_margin_forced`: if `true` will override the check of `exit_processor_sla_margin` against
+      `min_exit_period_seconds`
+    - `min_exit_period_seconds`: should reflect the value of this parameter for the specific child chain watched,
+    - `ethereum_block_time_seconds`: just to relate blocks to seconds for the `exit_processor_sla_margin` check
+    - `metrics_collection_interval`: how often are the metrics sent to `telemetry` (in milliseconds)
```

Comment:
Can we refactor `metrics_collection_interval` to `metrics_collection_interval_ms`. Sorry - we often use the suffix elsewhere.

Reply:
I'd love to, but this would be a global refactor, to keep consistency, so I'd rather do that separately.